src/main/java/me/crespel/runtastic/mapper/GpxSportSessionMapper.java

```diff
@@ -116,9 +182,52 @@ protected String mapSport(String sportTypeId) {
 			return "9"; // Running
 		case "3":
 			return "1"; // Biking
+		case "7":
+			return "Hiking"; // Hiking

```

Comment:
A GPX export from Strava gives me type 4 for Hiking. Not sure there's an official list for these...
```suggestion
			return "4"; // Hiking
```

Reply:
Requires more investigation, it seems not clear what/how to map.
mapSport(String sportTypeId) needs further adaptations.
E.g. for GPX output the sport type can be added in plain text.
See also https://www.topografix.com/gpx_manual.asp#type

Reply:
Tracked with https://github.com/fcrespel/runtastic-export-converter/issues/6